### PR TITLE
Update steps.rb

### PR DESCRIPTION
**Pull request by lbernardelli**
Adding a possibility to set multiple headers configurations.

### CODE REVIEW

File: lib/cucumber-api/steps.rb [hunk 1]
```diff
@@ -21,6 +21,16 @@
   }
 end
 
+Given(/^I set Headers to:$/) do |params|
+  @headers = {
```
Comment (hidroh) [inline lib/cucumber-api/steps.rb]: I prefer to remove this default `@headers` initialization, leave it to the existing steps, and change this step def to `I add Headers:`. This way you can use this in conjunction with current steps like:

```
I send ? and accept JSON
And I add Headers:
   | name | value |
```

**Review by hidroh — CHANGES_REQUESTED** [review]
Can you also add a sample usage in README? Thanks!

**Comment by lbernardelli** [comment]
I think it's ok now. I got the hook and provided an example to sample feature. Thanks to take a look on this. :+1:

**Review by hidroh — APPROVED** [review]
LGTM

**Pull request merged**